### PR TITLE
[OF-1233] feat: add OrganizationId to Space

### PR DESCRIPTION
**Pull request by MAG-AW**
connected-spaces-platform Pull Request

**For the reviewer**

* [ ] If required, are the changes covered by appropriate tests?
* [ ] Are any public-facing API changes well documented?
* [ ] Is the code easily readable and extensible and/or follow existing conventions?

### CODE REVIEW

File: Tests/src/PublicAPITests/OrganizationSystemTests.cpp [hunk 1]
```diff
@@ -436,3 +436,44 @@ CSP_PUBLIC_TEST(CSPEngine, OrganizationSystemTests, BulkInviteToOrganizationTest
...
+
+	// Log in - user needs to be an admin member of the organization
+	String AdminUserId;
+	LogIn(UserSystem, AdminUserId, OrgAdminUserEmail, OrgAdminUserPassword);
+
+	// Create space
+	::Space Space;
+	CreateSpace(SpaceSystem, UniqueSpaceName, TestSpaceDescription, SpaceAttributes::Private, nullptr, nullptr, nullptr, Space);
+
+	EXPECT_EQ(Space.OrganizationId, "661ea67b07a518c46909dc97");
```
Comment (MAG-MichaelK) [inline Tests/src/PublicAPITests/OrganizationSystemTests.cpp:471]: Hard-coded IDs are never good. Can we store this with the login details instead?

File: Tests/src/PublicAPITests/OrganizationSystemTests.cpp [hunk 2]
```diff
@@ -436,3 +436,44 @@ CSP_PUBLIC_TEST(CSPEngine, OrganizationSystemTests, BulkInviteToOrganizationTest
 	LogOut(UserSystem);
 }
 #endif
+
+#if RUN_ALL_UNIT_TESTS || RUN_ORGANIZATIONSYSTEM_TESTS || RUN_ORGANIZATIONSYSTEM_CREATE_ORGANISATION_SPACE_TEST
+CSP_PUBLIC_TEST(CSPEngine, OrganizationSystemTests, CreateOrganisationSpaceTest)
+{
+	auto OrgAccountDetails	  = GetOrganizationAccountDetails();
+	bool HasOrgAccountDetails = OrgAccountDetails.HasKey("OrgAdminLoginEmail") && OrgAccountDetails.HasKey("OrgAdminLoginPassword");
+	EXPECT_TRUE(HasOrgAccountDetails);
```
Comment (MAG-MichaelK) [inline Tests/src/PublicAPITests/OrganizationSystemTests.cpp:445]: Really nit-picky, but can we get a new line before this, please?

**Review by MAG-MichaelK — CHANGES_REQUESTED** [review]

File: Tests/src/PublicAPITests/OrganizationSystemTests.cpp (hunk 1 above)
Comment (MAG-MichaelK) [reply Tests/src/PublicAPITests/OrganizationSystemTests.cpp:471]: Better yet, can we retrieve this from CHS at runtime?

File: Tests/src/PublicAPITests/OrganizationSystemTests.cpp [hunk 3]
```diff
@@ -436,3 +442,51 @@ CSP_PUBLIC_TEST(CSPEngine, OrganizationSystemTests, BulkInviteToOrganizationTest
...
+	auto* OrganizationSystem = SystemsManager.GetOrganizationSystem();
+	auto* SpaceSystem		 = SystemsManager.GetSpaceSystem();
+
+	const char* TestSpaceName		 = "OLY-UNITTEST-SPACE-REWIND";
+	const char* TestSpaceDescription = "OLY-UNITTEST-SPACEDESC-REWIND";
+
+	char UniqueSpaceName[256];
+	SPRINTF(UniqueSpaceName, "%s-%s", TestSpaceName, GetUniqueString().c_str());
+
+	// Log in - user needs to be an admin member of the organization
```
Comment (MAG-AdamThorn) [inline Tests/src/PublicAPITests/OrganizationSystemTests.cpp:470]: They don't need to be an admin. Can you move the comment after the '-' please?

**Review by MAG-AdamThorn — CHANGES_REQUESTED** [review]

**Review by MAG-AdamThorn — APPROVED** [review]

**Pull request merged**